### PR TITLE
Update URL for definition of 'web page' on HTML page has non empty title

### LINKED ISSUE #1993: "HTML page has non-empty title" (2779a5) should use our definition of "web page"
I just noticed that [HTML page has non-empty title](https://www.w3.org/WAI/standards-guidelines/act/rules/2779a5/) is still linking to the [WCAG definition of web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) rather than [ours](https://act-rules.github.io/glossary/#web-page-html).
Not sure this is unambiguous enough 🤔

### PR DESCRIPTION
**Pull request by sboptum**
This is a change to the URL for 'web page' listed under Applicability. The current link is to the WCAG definition, rather than the ACT-Rules Community's, which is clearer, also links to the WCAG definition, and contains links to the relevant rules.

Closes issue(s):
https://github.com/act-rules/act-rules.github.io/issues/1993

- closes #1993)

Need for Call for Review:
This can be merged with 1 approval << editorial changes to website/test code

---

### **When creating PR:**

- [ x] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ x] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [x ] Add yourself (and co-authors) as "Assignees" for PR.
- [x ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ x] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ x] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/html-page-non-empty-title-2779a5.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element.
+This rule applies to the root element of the [web page](https://act-rules.github.io/glossary/#web-page-html), if it is an `html` element.
```
Comment (carlosapaduarte) [inline _rules/html-page-non-empty-title-2779a5.md]:
```suggestion
This rule applies to the root element of the [web page](#web-page-html), if it is an `html` element.
```

We have scripts in the backend that generate the full link to the glossary entry, depending on the web site that is being generated (https://act-rules.github.io/rules/2779a5 or https://www.w3.org/WAI/standards-guidelines/act/rules/2779a5/).  Therefore, we should not add the absolute path URL, but instead just reference the key for the glossary entry.

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Need to update the reference link

**Comment by sboptum** [comment]
Thanks, @carlosapaduarte. I understand the change and have accepted the edit.

**Review by carlosapaduarte — APPROVED** [review]

**Review by HelenBurge — APPROVED** [review]

File: _rules/html-page-non-empty-title-2779a5.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element.
+This rule applies to the root element of the [web page](#web-page-html), if it is an `html` element.
```
Comment (Jym77) [inline _rules/html-page-non-empty-title-2779a5.md]: I think the link needs to be defined in the definition list in order to be picked up when building the page 🤔 

* At the bottom of the file, add the definition `[web page]: #web-page-html 'Definition of HTML web page'` (next to the existing definition, on a separate line).
* Here, simply reference it with `[web page][]`.

**Review by Jym77 — CHANGES_REQUESTED** [review]
Need to improve a bit the Markdown syntax.

**Review by sboptum — COMMENTED** [review]
@Jym77 Made these changes and added the web page definition to the bottom of the file.

File: _rules/html-page-non-empty-title-2779a5.md (hunk 2 above)
Comment (sboptum) [reply _rules/html-page-non-empty-title-2779a5.md]: Change has been made. Please review and let me know if I missed it. Thank you.

**Comment by sboptum** [comment]
@Jym77 Can you approve this yet?

**Review by Jym77 — APPROVED** [review]

**Pull request merged**